tools/tune/drc/example_drc.m

```diff
@@ -1,18 +1,42 @@
 function example_drc();
 
 % Set the parameters here
-tplg_fn = "../../topology/m4/drc_coef_default.m4" % Control Bytes File
+tplg_fn = "../../topology/m4/drc_coef_default.m4"; % Control Bytes File
 % Use those files with sof-ctl to update the component's configuration
-blob_fn = "../../ctl/drc_coef.blob" % Blob binary file
-alsa_fn = "../../ctl/drc_coef.txt" % ALSA CSV format file
+blob_fn = "../../ctl/drc_coef.blob"; % Blob binary file
+alsa_fn = "../../ctl/drc_coef.txt"; % ALSA CSV format file
 
 endian = "little";
 
-% DRC enabled
-enabled = 1
+sample_rate = 44100;
```

Comment:
Probably 48 kHz is better assumption for SOF?

Reply:
Yes and we use 48000 in CRAS, this should be changed